R/create_run_dev.R

```diff
@@ -0,0 +1,97 @@
+#' Create run_dev by module
+#' 
+#' Allow you to create one run_dev by module
+#'
+#' @param name name of the script
+#' @param module_ui name of the module UI
+#' @param module_server name of the module server
+#' @param ... add parameters to your module_server. They have to be a string like param = "'jean'"
+#' @param pkg path to package
+#' @param open boolean open the file
+#'
+#' @export
+#'
+#' @examples
+#' 
+#' \dontrun{
+#' create_run_dev("ok", 
+#'                mod_ok_ui,
+#'                mod_ok_server, 
+#'                r= "reactiveValues(message = 'ok')", 
+#'                message = "'have to be string'")
+#' }
+create_run_dev <- function(name,
```

Comment:
Can you rename that add_run_dev_module so it's consistent with the rest of the naming convention?

Reply:
Sorry to push directly into master, I forgot that we have a branch dev on golem :D. But i write this part on golem book.... tough life... :)